<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Med</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="selectedProcess">
          <property name="toolTip">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Click Process to select a target process</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="process">
          <property name="text">
           <string>&amp;Process</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="pauseCheckbox">
          <property name="toolTip">
           <string>Pause process</string>
          </property>
          <property name="text">
           <string>Pa&amp;use</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="scanEntry"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lastDigit">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Last Digit</string>
          </property>
          <property name="placeholderText">
           <string>Last Digit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="found">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Number of address found</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="scanType">
          <item>
           <property name="text">
            <string>int8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>int16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>int32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>int64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>float32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>float64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>string</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>custom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ptr32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ptr64</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="scanButton">
          <property name="text">
           <string>&amp;Scan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterButton">
          <property name="text">
           <string>F&amp;ilter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTreeView" name="scanTreeView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="scopeStart">
               <property name="placeholderText">
                <string>Scope start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="scopeEnd">
               <property name="placeholderText">
                <string>Scope end</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="scanAdd">
               <property name="toolTip">
                <string>Add the selected address to Store</string>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="scanAddAll">
               <property name="toolTip">
                <string>Add all to Store</string>
               </property>
               <property name="text">
                <string>Add All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="scanClear">
               <property name="toolTip">
                <string>Clear addresses</string>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QPlainTextEdit" name="notes">
          <property name="placeholderText">
           <string>Notes are written here</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeView" name="storeTreeView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="prevAddress">
              <property name="toolTip">
               <string>Create previous address</string>
              </property>
              <property name="text">
               <string>Previous</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="nextAddress">
              <property name="toolTip">
               <string>Create next address</string>
              </property>
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLineEdit" name="shiftFrom">
              <property name="placeholderText">
               <string>Shift from</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="shiftTo">
              <property name="toolTip">
               <string>Target shift address OR number of bytes to be moved</string>
              </property>
              <property name="placeholderText">
               <string>Shift to / bytes</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="storeShift">
              <property name="toolTip">
               <string>Shift selected addresses</string>
              </property>
              <property name="text">
               <string>Shift</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="storeUnshift">
              <property name="toolTip">
               <string>Unshift selected addresses</string>
              </property>
              <property name="text">
               <string>Unshift</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moveAddress">
              <property name="toolTip">
               <string>Move the selected by number of bytes (+/-)</string>
              </property>
              <property name="text">
               <string>Move</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QComboBox" name="namedScans">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Named scans</string>
        </property>
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="namedScan_delete">
        <property name="toolTip">
         <string>Delete current named scan</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="namedScan_name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The name scan to be added</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="namedScan_add">
        <property name="toolTip">
         <string>Add new named scan</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionReload"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuStore">
    <property name="title">
     <string>A&amp;ddress</string>
    </property>
    <addaction name="actionNewAddress"/>
    <addaction name="actionDeleteAddress"/>
    <addaction name="actionStoreClear"/>
    <addaction name="separator"/>
    <addaction name="actionMemEditor"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menu_Encoding">
     <property name="title">
      <string>&amp;Encoding</string>
     </property>
     <addaction name="actionDefaultEncoding"/>
     <addaction name="actionBig5Encoding"/>
    </widget>
    <addaction name="actionRefresh"/>
    <addaction name="actionAutoRefresh"/>
    <addaction name="actionResumeProcess"/>
    <addaction name="actionShowNotes"/>
    <addaction name="separator"/>
    <addaction name="menu_Encoding"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>&amp;Option</string>
    </property>
    <addaction name="actionFastScan"/>
    <addaction name="actionForceResume"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOption"/>
   <addaction name="menuStore"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>&amp;Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Sa&amp;ve</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNewAddress">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionShiftAll">
   <property name="text">
    <string>Shift All</string>
   </property>
  </action>
  <action name="actionDeleteAddress">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionMemEditor">
   <property name="text">
    <string>&amp;Editor</string>
   </property>
  </action>
  <action name="actionShowNotes">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Notes</string>
   </property>
  </action>
  <action name="actionDefaultEncoding">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Default</string>
   </property>
  </action>
  <action name="actionBig5Encoding">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Big5</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAutoRefresh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Refresh</string>
   </property>
  </action>
  <action name="actionResumeProcess">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Resume Process</string>
   </property>
  </action>
  <action name="actionFastScan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fast Scan</string>
   </property>
  </action>
  <action name="actionStoreClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionForceResume">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>F&amp;orce Resume</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>mainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
